frontend/packages/akr/src/tests/cypress/integration/public_translator.spec.ts

```diff
@@ -30,6 +30,15 @@ describe('PublicTranslatorFilters', () => {
     onPublicTranslatorFilters.expectSeachBtnText('Näytä tulokset (1)');
   });
 
+  it('should return results only with the exact town name when filtering by town name', () => {
+    onPublicTranslatorFilters.expectSeachBtnText('Näytä tulokset (50)');
+
+    onPublicTranslatorFilters.filterByTown('Kemi');
+
+    onPublicTranslatorFilters.expectSeachBtnText('Näytä tulokset (1)');
+    onPublicTranslatorsListing.expectTranslatorsCount(1);
```

Comment:
Lisäiskö vast. testinä myös filteröinnin ja tulosten tsekkauksen 'Kemiö' hakusanalla? Näin ei riippuis testin läpimeno siitä, onko fixturessa Kemiö vai ei ole

Reply:
nyt tehdään tsekkaus 'Kemiö' hakusanalla myös.